cli/js/lib.deno.ns.d.ts

```diff
@@ -155,6 +155,8 @@ declare namespace Deno {
    */
   export function execPath(): string;
 
+  export function osType(): string;
```

Comment:
Hi - thanks for the patch - but in order to add new APIs they need to first be unstable. Please move this declaration to cli/js/lib.deno.unstable.d.ts

Reply:
done